src/shared/ui/section-label/section-label.test.tsx

```diff
@@ -21,4 +21,39 @@ describe('SectionLabel component', () => {
 
     expect(label).toBeInTheDocument();
   });
+
+  it('applies default variants correctly', () => {
```

Comment:
Since we have UI tests I don't think we need to check for the presence of classes that define styles only

Reply:
> Since we have UI tests I don't think we need to check for the presence of classes that define styles only

Okok, i can remove this test cases.